ops/log.py

```diff
@@ -26,7 +26,21 @@ def emit(self, record):
         self.model_backend.juju_log(record.levelname, self.format(record))
 
 
-def setup_root_logging(model_backend):
+def setup_root_logging(model_backend, debug=False):
+    """Setup python logging to forward messages to juju-log.
+
+    model_backend -- a ModelBackend to use for juju-log
+    debug -- if True, enable DEBUG level logging, and write logs to stderr as well as to juju-log.
+    """
+    if debug:
+        level = logging.DEBUG
+    else:
+        level = logging.INFO
```

Comment:
This is fine, but remember you can always write `level = logging.DEBUG if debug else logging.INFO`

Reply:
I do realize that newer python has the Ternary operator, but I don't find that it is actually easier to read. The problem is that (foo if blah else bing) doesn't have great punctuation to clarify what the result is. I suppose the 2 possibilities are at the very beginning or the very end.